src/main.rs

```diff
@@ -20,22 +20,38 @@ enum Commands {
     On {
         #[clap(long, short, help = "The serial number of the Logitech Litra device")]
         serial_number: Option<String>,
+        #[clap(long, help = "Select devices by their type (LitraGlow, LitraBeam, LitraBeamLX)")]
```

Comment:
I think we should add a validation when running all the commands to ensure that exactly zero or one of these filters is applied, and not multiple. I think that will be easier to reason about.